parent/pom.xml

```diff
@@ -284,7 +284,7 @@
                 <repository>
                     <id>sonatype-nexus-staging</id>
                     <name>Sonatype Nexus Staging</name>
-                    <url>https://oss.sonatype.org/content/repositories/staging</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/staging</url>
```

Comment:
Same comment as above... Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?

Reply:
This is "staging", not "snapshots".

But yes, why not define a property for consistency and use it.